kobo/apps/superuser_stats/views.py

```diff
@@ -105,10 +105,11 @@ def continued_usage_report(request):
         f'To select a date range, add a <code style="background: lightgray">?</code> at the end of the URL and set the '
         f'<code style="background: lightgray">end_date</code> parameter to <code style="background: lightgray">YYYY-MM-DD</code>.<br><br>'
         f'<b>Example:</b><br>'
-        f'<a href="{url}?end_date={tomorrow}">'
-        f'  {url}?end_date={tomorrow}'
+        f'<a href="{url}?end_date={tomorrow.date()}T{tomorrow.hour}:{tomorrow.minute}:{tomorrow.second}">'
+        f'  {url}?end_date={tomorrow.date()}T{tomorrow.hour}:{tomorrow.minute}:{tomorrow.second}'
         f'</a>'
-        f'<p>The default end date is {tomorrow}.</p>'
+        f'<p>The default end date and time is '
+        f'{tomorrow.date()}T{tomorrow.hour}:{tomorrow.minute}:{tomorrow.second}.</p>'
```

Comment:
@JacquelineMorrissette I think it would be nicer to have the date filtering be the same format as the other exports with filtering, i.e. not requiring the time from the user. Maybe this can just be appended after the fact if that's the main issue?